solidity/dashboard/src/utils/arithmetics.utils.js

```diff
@@ -33,6 +33,12 @@ export const lte = (a, b) => {
 
 export const isZero = (a) => web3Utils.toBN(a).isZero()
 
+export const calculatePercentage = (percentage, value) => {
```

Comment:
Can we rename `value` to `total`? because we calculate a value from percentage so we need total value. Also wondering about a better name for this function.

Reply:
Since we calculate a value **from** a **percentage** of another number then maybe we could just name it as `fromPercentage`? 🤔 Can't think of a better name right now 😄

Reply:
> Can't think of a better name right now
me too 😆

I think `fromPercentage` it's better than `calculatePercentage`. So let's rename it to `fromPercentage`.